services/broker/.lagoon.app.yml

```diff
@@ -120,6 +120,8 @@ objects:
                   key: RABBITMQ_PASSWORD
             - name: RABBITMQ_USE_LONGNAME
               value: "true"
+            - name: RABBITMQ_DEFAULT_HA_PATTERN
+              value: "^lagoon-"
```

Comment:
let's add this to the `services/broker/Dockerfile` instead of into the broker statefulset, as then it is part of the Docker Image already.

Reply:
So you prefer users let override the variable during the build of the image instead during the creation of StatefulSet. is it correct?

Reply:
how users overwrite it themselves I don't care, it's the users decision. But I would like that Lagoon when it uses the base images sets the ENV variables needed for itself to correctly work inside the Service Dockerfiles and not the YAML files.

Reply:
Added ENV variables to define default VHOST and default PATTERN in broker service